src/reachy2_sdk/subparts/lidar.py

```diff
@@ -0,0 +1,140 @@
+        self._safety_enabled = response.safety_on.value
+
+    @property
+    def safety_slowdown_distance(self) -> float:
+        """Safety distance in meters of the mobile base from obstacles.
+
+        The mobile base's speed is slowed down if the direction of speed matches the direction of
+        at least 1 LIDAR point in the safety_distance range.
+        """
+        return float(self._safety_distance)
+
+    @safety_slowdown_distance.setter
+    def safety_slowdown_distance(self, value: float) -> None:
+        self._stub.SetZuuuSafety(
+            LidarSafety(
+                safety_distance=FloatValue(value=value),
+                critical_distance=FloatValue(value=self._critical_distance),
+                safety_on=BoolValue(value=self._safety_enabled),
+            )
+        )
+        self._update_safety_info()
+
+    @property
+    def safety_critical_distance(self) -> float:
+        """Critical distance in meters of the mobile base from obstacles.
+
+        The mobile base's speed is changed to 0 if the direction of speed matches the direction of
+        at least 1 LIDAR point in the critical_distance range.
+        If at least 1 point is in the critical distance, then even motions that move away from the obstacles are
+        slowed down to the "safety_zone" speed.
+        """
+        return float(self._critical_distance)
+
+    @safety_critical_distance.setter
+    def safety_critical_distance(self, value: float) -> None:
+        self._stub.SetZuuuSafety(
+            LidarSafety(
+                safety_distance=FloatValue(value=self._safety_distance),
+                critical_distance=FloatValue(value=value),
+                safety_on=BoolValue(value=self._safety_enabled),
+            )
+        )
+        self._update_safety_info()
+
+    @property
+    def safety_enabled(self) -> bool:
+        """Enable or disable the safety feature."""
+        return self._safety_enabled
+
+    @safety_enabled.setter
+    def safety_enabled(self, value: bool) -> None:
+        self._stub.SetZuuuSafety(
+            LidarSafety(
+                safety_distance=FloatValue(value=self._safety_distance),
+                critical_distance=FloatValue(value=self._critical_distance),
+                safety_on=BoolValue(value=value),
+            )
+        )
+        self._update_safety_info()
+
+    @property
+    def obstacle_detection_status(self) -> LidarObstacleDetectionStatus:
+        """Get status of the lidar obstacle detection.
+
+        Can be either NO_OBJECT_DETECTED, OBJECT_DETECTED_SLOWDOWN, OBJECT_DETECTED_STOP or DETECTION_ERROR.
+        """
+        return self._obstacle_detection_status
+
+    def reset_safety_default_values(self) -> None:
+        """Reset default distances values for safety detection.
+
+        Reset values are:
+        - safety_critical_distance
+        - safety_slowdown_distance.
+        """
+        self.safety_critical_distance = 0.55
```

Comment:
See comment on init. We can set a "constant" self.SAFETY_CRITICAL_DISTANCE_DEFAULT = 0.55 and use it here and in the init

Reply:
Same, all is correctly init in #294